ci/azure-pipelines.yml

```diff
@@ -78,76 +78,8 @@ stages:
               invoke lint || EXIT_STATUS=$?
               exit $EXIT_STATUS
 
-  - stage: custom_checks
-    pool:
-      vmImage: 'ubuntu-latest'
-
-    jobs:
-    - job: static_type_check
-      condition: or(eq(variables.isScheduled, true), eq(variables.isReleasePrep, true), eq(variables.isRelease, true), eq(variables.isManual, true))
-      steps:
-      - task: UsePythonVersion@0
-        inputs:
-          versionSpec: 3.8
-        displayName: 'Use Python 3.8'
-      - script: |
-          python -m pip install --upgrade pip
-          pip install --requirement requirements-types.txt
-        name: InstallDependencies
-      - script: |
-          invoke type-check --ci --pretty
-        name: StaticTypeCheck
-      - script: |
-          # initial run doesn't check `.pyi` source files
-          invoke type-check --ci --pretty --check-stub-sources
-        name: TypeCheckStubSourceFiles
-    - job: docstring_linter
-      condition: or(eq(variables.isScheduled, true), eq(variables.isReleasePrep, true), eq(variables.isRelease, true), eq(variables.isManual, true))
-      steps:
-      - script: |
-          pip install --requirement reqs/requirements-dev-test.txt
-          invoke docstrings
-        name: DocstringLinter
-
-    - job: unused_import_checker
-      condition: or(eq(variables.isScheduled, true), eq(variables.isReleasePrep, true), eq(variables.isRelease, true), eq(variables.isManual, true))
-      steps:
-      - script: |
-          pip install $(grep -E '^(ruff)' reqs/requirements-dev-contrib.txt)
-          # https://www.flake8rules.com/rules/F401.html - Prunes the dgtest graph to improve accuracy
-          ruff --select F401 great_expectations tests
```

Comment:
Does this test get added? I don't see it in github actions.

Reply:
We check for this as part of the normal linting step.

Reply:
You mean F401 is checked by default by ruff? I wonder why we had a separate step here?